src/monitor.tf

```diff
@@ -0,0 +1,21 @@
+resource "azurerm_resource_group" "monitor_rg" {
```

Comment:
you can copy from here
https://github.com/pagopa/cstar-infrastructure/blob/main/src/monitor.tf